### PR TITLE
Add kwargs to support recent Xonsh

### PR DESCRIPTION
**Pull request by greg-hellings**
Changes needed to let the code load on modern Xonsh versions. I have also updated README and others with the intention to fork the repo if @74th is no longer interested in maintaining it.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -4,17 +4,22 @@ https://direnv.net
 
 https://xon.sh
 
-This works with python 3.7.0, xonsh 0.8.8 and direnv 2.17.0.
+This works with python 3.11.0, xonsh 0.17.0 and direnv 2.34.0.
 
 ::
 
-	xpip install xonsh-direnv
+	xpip install "git+https://github.com/greg-hellings/xonsh-direnv@master"
```
Comment (con-f-use) [inline README.rst:11]: Not a good idea, the official readme should have people install and actual release from pypi.

File: setup.py [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
+    version="1.7.0",
     license="MIT",
-    url="https://github.com/74th/xonsh-direnv",
+    url="https://github.com/greg-hellings/xonsh-direnv",
     description="direnv support for the xonsh shell",
     long_description=long_description,
-    author="74th",
-    author_email="site@74th.tech",
+    author="Greg Hellings",
+    author_email="greg@thehellings.com",
```
Comment (con-f-use) [inline setup.py:14]: So you are taking control of the project and are retroactively the author of all of it now?

If you really want to become a maintainer, you should just ask beforehand, make the changes in a separate PR and rather than replacing the author add yourself as maintainer. Or generally don't sneak around like this.

**Comment by 74th** [comment]
Thank you for your contribution to the xonsh-direnv project. I appreciate the effort you have put into updating the code to support recent versions of Xonsh. Your changes seem necessary and beneficial for the project.

However, I am a bit confused about the part of the PR that mentions rewriting the author. While I understand the need for updates, changing the author information is something that requires careful consideration.

Regarding the maintenance of the project, I must admit that my interest in xonsh-direnv has waned, and I am open to having someone else take over its maintenance. I also understand that for this to be meaningful, the PyPI ownership would need to be transferred as well. However, I am not yet sure if you are the right person to hand over this responsibility to.

Could you please provide some information about your experience with maintaining similar projects? This will help me feel more confident about making the transition.

Thank you for your understanding and for your willingness to contribute to the project.

**Comment by con-f-use** [comment]
Closed in favor of #18 - @greg-hellings please open an issue or separate PR to discuss maintainership.

**Comment by greg-hellings** [comment]
Since the repository hadn't been updated in 2 years I was just preparing in case you had lost willingness or interest in maintaining the project. I've lost count of the number of times I've sent in PRs on something a couple of years old that just sat ignored. It looks like I misjudged that, so I'm happy to leave it with you. Thanks for the quick turnaround on getting this fix in. I use this xontrib pretty regularly switching between Python projects, work repositories, and so forth. So I'm glad to see it still usable and maintained.
